tests/rules/test_su.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from thefuck.rules.su import match, get_new_command
+from thefuck.types import Command
+
+
+@pytest.mark.parametrize('output', [
+    'command not found: sudo'])
+def test_match(output):
+    assert match(Command('', output))
```

Comment:
It would be good having some practical test cases. `command.script_parts` is part of the `match` condition but not taken into consideration in tests.